app/views/devise/confirmations/new.html.haml

```diff
@@ -0,0 +1,10 @@
+%h2 Resend confirmation instructions
```

Comment:
please remove all devise views that you do not change in any way

Reply:
All views out of this feature's scope removed.